pkg/domain/nexpose/asset.go

```diff
@@ -0,0 +1,272 @@
+package nexpose
```

Comment:
These are mostly copied from generated Go code from the Nexpose Open API spec. I realize they _all_ have `omitempty`, but the API doesn't really explicitly define what all is required.

Reply:
omitempty doesn't have too much of an effect when unmarshaling from JSON, it's more for marshaling to JSON. If the JSON field doesn't exist, the zero value for the go type will get set. This would behave nearly the same if the empty value got unmarshaled into the go struct.